src/message.rs

```diff
@@ -1384,6 +1394,69 @@ pub async fn mdn_from_ext(
     None
 }
 
+/// Marks a message as failed after an ndn (non-delivery-notification) arrived.
+/// Where appropriate, also adds an info message telling the user which of the recipients of a group message failed.
+pub(crate) async fn ndn_from_ext(
```

Comment:
What does `ext` stands for? Extract, extended?

Reply:
@link2xt, non-delivery-notification from external? I actually only named it this way because the mdn-parsing fn is called mdn_from_external

Reply:
But we could rename them to handle_ndn() and handle_mdn(). (?)

Reply:
After some commit digging, found that it was named `mrmailbox_readreceipt_from_ext__` at some point, but no idea what `ext` stands for there: https://github.com/deltachat/deltachat-core/commit/b4fa3059d811773a06f4157ea16960fa734fe727

So I'd remove this `ext` as it is meaningless now.

Reply:
i think this `ext` was for "external", does not make sense. not sure what my thoughts were that time :)

Reply:
Done.